airtable/plugin.py

```diff
@@ -83,21 +113,22 @@ def _resolve_error_message(
 
 @dataclass
 class AirtableRecord:
-    id: str
+    airtable_record_id: AirtableRecordID
     created_time: datetime
     fields: dict[str, Any]
 
 
-def airtable_record_list(base_id: str, table_id_or_name: str) -> list[AirtableRecord]:
+def airtable_record_list(airtable_base_id: AirtableBaseID, airtable_table_id: AirtableTableID) -> list[AirtableRecord]:
```

Comment:
Similar question here; is `base_id` + type hint sufficient?

Reply:
+1 no need to prefix arg names, especially if there's a type for the arg that's good